src/pages/taekjun/Login/taekjunLogin.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import './taekjunLogin.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: '',
+      pwValue: '',
+    };
+  }
+
+  getValue = e => {
+    const { name, value } = e.target;
+    if (name === 'id') {
+      value.indexOf('@') !== -1
+        ? this.setState({
+            idValue: value,
+          })
+        : this.setState({ idValue: value });
+    }
+
+    if (name === 'pw') {
+      value.length > 4
+        ? this.setState({
+            pwValue: value,
+          })
+        : this.setState({ pwValue: value });
```

Comment:
```suggestion
    const { name, value } = e.target;
    this.setState({[name]:value});
```
name 값을 id, pw가 아니라 state의 key값과 동일하게 idValue, pwValue로 설정하고 계산된 속성명 이용하면 위와 같이 표현 가능합니다.

Reply:
이 부분은 이해가 잘 안 됩니다. 월요일에 여쭤보고 수정해도 될까요?